src/main/java/Duke.java

```diff
@@ -1,10 +1,232 @@
+import exception.DukeException;
+import exception.ErrorMessage;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.ObjectOutputStream;
+import java.io.ObjectInputStream;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+    private static void logoWithHello() {
```

Comment:
Consider a better (more generic) name than `logoWithHello` or `printLogoWithHello`.
What if you eventually didn't print hello anywhere?
Target the operation, rather than the specifics